regression_tests/regression_test.go

```diff
@@ -0,0 +1,120 @@
+//go:build regression
+// +build regression
+
+package regressiontests
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	fundsPerTest                  = 2
+	adminXPriv                    = "xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK"
+	adminXPub                     = "xpub661MyMwAqRbcFgfmdkPgE2m5UjHXu9dj124DbaGLSjaqVESTWfCD4VuNmEbVPkbYLCkykwVZvmA8Pbf8884TQr1FgdG2nPoHR8aB36YdDQh"
+	errorWhileGettingTransaction  = "error while getting transaction: %s"
+	errorWhileCreatingUser        = "error while creating user: %s"
+	errorWhileGettingBalance      = "error while getting transaction: %s"
+	errorWhileSendingFunds        = "error while sending funds: %s"
+	errorWhileGettingSharedConfig = "error while getting shared config: %s"
+	errorWhileGettingEnvVariables = "error while getting env variables: %s"
+)
+
+var (
+	clientOneURL         string
+	clientTwoURL         string
+	clientOneLeaderXPriv string
+	clientTwoLeaderXPriv string
+)
+
+func TestRegression(t *testing.T) {
```

Comment:
`ctx := context.Background` and then pass it to needed helper functions

Reply:
Now ctx is declared here.